src/ImgixTag.js

```diff
@@ -191,13 +191,23 @@ var ImgixTag = (function () {
     const el = this.el;
     const _window = this.window;
 
-    if (existingSizes || ixSizes !== 'auto') {
-      return existingSizes != null ? existingSizes : '100vw';
-    } else if (ixSizes === 'auto') {
+    /**
+     *
+     * The conditionals bellow decide when to override the value for `sizes` for
+     * the given element.
+     *
+     * - If `sizes` set, we leave the value as is even if `ix-sizes` is `auto`
+     * - If `sizes` not set and `ix-sizes` not auto, set `sizes` to `ix-sizes`
```

Comment:
Shall we add a test case for this?

Reply:
Sorry, just to be clear, do you mean we should add a test for:
>  "If `sizes` not set and `ix-sizes` not auto, set `sizes` to `ix-sizes`"

I believe I've already [added](https://github.com/imgix/imgix.js/blob/fba7ed4f23a1fd564f1f036eefea8435b2b43909/cypress/integration/sizesAutoSpec.js#L9-L16) a test for:
>  "If `sizes` set, we leave the value as is even if `ix-sizes` is `auto`"

Reply:
That's correct